src/handlers/check_commits.rs

```diff
@@ -350,3 +367,139 @@ r#":warning: **Warning** :warning:
      - 123456789"#
     );
 }
+
+#[cfg(test)]
```

Comment:
Maybe wrap this whole thing into `#[cfg(test)] mod tests { ... }`, as we usually do?